github3/repos/repo.py

```diff
@@ -1282,6 +1282,32 @@ def imported_issue(self, imported_issue_id):
         return self._instance_or_null(ImportedIssue, json)
 
     @requires_auth
+    def imported_issues(self, number=-1, since=None, etag=None):
+        """Retrieve imported issues
+
+        :param int number: (optional), number of imported issues to return.
+            Default: -1 returns all branches
+        :param since: (optional), Only commits after this date will
```

Comment:
Ah. I hadn't double checked the gist. I'd update the docstring to say this then:

```rst
This can be a ``datetime`` instance, ISO8601 formatted date string, or a string formatted like so: ``2016-02-04``, i.e., ``%Y-%m-%d``.
```